independent-projects/tools/codestarts/codestarts.adoc

```diff
@@ -0,0 +1,220 @@
+= Quarkus - Codestarts
+
+This guide explains how to create and configure a Quarkus Codestart for an extension.
+
+== Description
+
+"Codestarts" is the name we gave to our Quarkus quickstart code generation system. 
+Codestarts provide a personalized Quarkus getting started experience and really show the Quarkus breadth.
+Quarkus extensions are able to provide one or more well defined codestarts which will contain the necessary resources and code examples required to get started using it.
+
+== How it works
+
+There are two kinds of codestarts contributing to the generation of a project, the kind where we want to have only one for a project (called Base) and the rest (called Extra).
```

Comment:
```suggestion
There are two kinds of codestarts contributing to the generation of a project, the kind where we want to have only one for the base project (called Base) and the rest (called Extra).
```

aren't they actually called 'base' and 'extra' no capitalization?

Reply:
yes I'll update